src/v2/App.jsx

```diff
@@ -14,8 +14,25 @@
 // limitations under the License.
 // -----------------------------------------------------------------------------
 
-import React from 'react'
+import React, {useEffect} from 'react'
+
+import AppFooter from './components/AppFooter.jsx'
+import PotentialFireList from './components/PotentialFireList.jsx'
+
+import './App.css'
 
 export default function App() {
-  return <div/>
+  // XXX: Reset scroll position on page load. Otherwise, the window may be
+  // scrolled a couple hundred pixels down (not sure why).
+  useEffect(() => {
+    ;(function check() {
+      /complete/.test(document.readyState) ? window.scrollTo(0, 0) : setTimeout(check)
+    })()
+  })
+
+  return 0,
```

Comment:
For JSX, it’s equivalent to:

```js
return (
  <div>
    ...
  </div>
)
```

Both of these conventions are getting around the fact that the following code doesn’t do what you’d want/expect:

```js
return
  <div>
    ...
  </div>
```

How `return 0, ...` works is that only the last value in a sequential expression is returned so the `0,` is ignored and ultimately transpiled out anyway.

`return 0, ...` is obviously my quirk since JSX’s inventors went with `return (...)`, but I find it (a) easier to type and (b) easier to read, and (c) frankly more elegant.